tree/treeplayer/inc/TTreeIndex.h

```diff
@@ -70,7 +70,8 @@ class TTreeIndex : public TVirtualIndex {
    virtual void           UpdateFormulaLeaves(const TTree *parent);
    virtual void           SetTree(const TTree *T);
 
-   ClassDef(TTreeIndex,2);  //A Tree Index with majorname and minorname.
+   //A Tree Index with majorname and minorname.
+   ClassDef(TTreeIndex,2)
```

Comment:
@couet @ferdymercury Can you verify that this type of change preserve the value of TClass::GetTitle:
```
root [1] TClass::GetClass("TTreeIndex")->GetTitle()
(const char *) "A Tree Index with majorname and minorname."
```

Reply:
@ferdymercury can you verify that as your root version has this change ?

Reply:
Hmm I suspect that it won't preserve it, but I will check it. Doxygen doesn't seem to like however the original formulation.

If it does not preserve it: would it be ok to rename those macros to:
`ClassDefTitle(TTreeIndex,2,"A Tree Index with majorname and minorname")`

and to store the title directly via this new macro, rather than via a comment ? (It would be more type-safe I guess).

Reply:
Never mind, it does not preserve it, so will revert those changes.